### PR TITLE
843 xsss amelioration ux wording et organisation des settings securite

### LINKED ISSUE #843: [XSSS] Amélioration UX, wording et organisation des settings sécurité.
Suite à un travail d'analyse du parcours XSSS mono-device sur iOS, voici une proposition de parcours amélioré.
XSSS-Monodevice-iOS-Proposition

Ici un résumé des modification à faire.

### 1. Changer l'ordre des sections des settings Sécurité.
- [ ] Nouvel ordre:
1. Sauvegarde automatique des message
2. Mes sessions
3. Signature croisée
4. Cryptographie
5. Pin & Face ID

### 2. Changer les noms des sections settings Sécurité.
- [ ] Mes sessions > **Appareils connectés**
- [ ] Cryptographie > **Clés Tchap**

### 3. Changer le wording de la section Sauvegarde automatique:
 Si activée: 
- [ ] picto _checkmark.circle.fill_ vert + "Activée: cet appareil sauvegarde vos messages".

Si pas activée:
- [ ] Etat: picto _xmark.circle.fill_ gris + "Desactivée: cet appareil ne sauvegarde pas vos messages".
- [ ] Bouton: Configurer > **Activer**

**Avant**
IMG_5663

**Après**
XSSS-Settings

### 3bis. Changer le wording de la section Signature Croisée:
 Si activée: 
- [ ] picto _checkmark.circle.fill_ vert + "Activée sur cet appareil".

Si pas activée:
- [ ] Etat: picto _xmark.circle.fill_ gris + "Desactivée sur cet appareil".
- [ ] Bouton: Configurer > **Activer**

### 4. Page génération code de Récupération
- [ ] Titre: "Notez votre Code de Récupération"
- [ ] Message "Notez ce code dans un endroit sûr.  Vous aurez à le copier / coller pour récupérer vos messages, donc évitez la capture d’écran."
- [ ] Bouton: "Copier le code"
- [ ] Ajouter sur le bouton un message "Attention : c'est la seule fois que votre code sera affiché !"
- [ ] Virer le bouton Continuer. L'appuis sur le bouton Copier le Code declanche directement la modale
- [ ] Titre de la modale: "Conservez-le en lieu sûr"

**Avant** 
IMG_5664

**Après**
Frame 2608887

### 5. Page récupération des message depuis device avec code
- [ ] Titre: Récupération des messages
- [ ] Bouton: Terminé > Terminer 
- [ ] (Si possible faire ce bouton comme le grand bouton de la création du Code)

**Avant** 
IMG_5666

**Après**
Frame 2608877

### 6. Modale suppresion de la sauvegarde
- [ ] Message: "En supprimant la sauvegarde automatique, vous risquez de perdre vos messages."

### 7. Modale activation de la Signature Croisée
- [ ] Titre en français: "Activer la signature croisée."

**Après**
Frame 2608889

### 8. Modale réinitialisation de la Signature Croisée
- [ ] Titre en français: "Êtes-vous sur ?"
- [ ] Message en français: "Faites cette opération seulement si vous avez perdu tous vos autres appareils vérifiés."
- [ ] Bouton en français: "Réinitialiser"

**Après**
Frame 2608888

### 9. Modale vérification appareil (suite à deconnexion)
- [ ] Message en français: "Une fois votre appareil vérifié, vous pourrez récupérer vos messages."

**Après**
Frame 2608884

### 10. Page vérification appareil (suite à deconnexion)
- [ ] Titre en français: "Vérifiez cet appareil"
- [ ] Message en français: "Vérifiez cet appareil pour récupérer vos messages. Si vous avez un autre appareil connecté à Tchap, ouvré l’application et suivez les instructions."
- [ ] Message bouton en français: "Vous n’avez pas d’autre appareil ?"
- [ ] Bouton: "Utilizer le Code de Récupération"

**Avant**
IMG_5673

**Après**
Frame 2608883

### 11. Page insertion Code de Récuperation
- [ ] Titre: "Récupération des messages"
- [ ] Message: "Utilisez votre Code de Récupération pour vérifier cet appareil."
- [ ] Bouton: "Utilizer le Code de Récupération"

**Avant**
IMG_5675

**Après**
Frame 2608885

### 12. Modale insertion mauvais Code de Récuperation
- [ ] Titre: "Code erroné"
- [ ] Message: "Vérifiez que vous avez saisi le Code de Récupération le plus récent.."

### PR DESCRIPTION
**Pull request by NicolasBuquet**
Fix #843 

Simulator Screen Shot - iPhone 14 - 2023-06-08 at 19 27 43
Simulator Screen Shot - iPhone 14 - 2023-06-08 at 19 28 24

### CODE REVIEW

File: Riot/Assets/en.lproj/Vector.strings [hunk 1]
```diff
@@ -1579,11 +1579,11 @@ Tap the + to start adding people.";
 "device_verification_self_verify_wait_new_sign_in_title" = "Verify this login";
 "device_verification_self_verify_wait_information" = "Verify this session from one of your other sessions, granting it access to encrypted messages.\n\nUse the latest %@ on your other devices:";
 "device_verification_self_verify_wait_additional_information" = "This works with %@ and other cross-signing capable Matrix clients.";
-"device_verification_self_verify_open_on_other_device_title" = "Open %@ on your other device";
-"device_verification_self_verify_open_on_other_device_information" = "You need to verify this session in order to read your secure message history.\n\nOpen Element on one of your other devices and follow the instructions.";
-"device_verification_self_verify_wait_recover_secrets_without_passphrase" = "Use Security Key";
+"device_verification_self_verify_open_on_other_device_title" = "Vérify this device"; // Tchap
```
Comment (odelcroi) [inline Riot/Assets/en.lproj/Vector.strings:1582]: ya un "é"

File: Riot/Assets/fr.lproj/Vector.strings [hunk 2]
```diff
@@ -1120,14 +1121,14 @@
...
+"secrets_recovery_with_key_title" = "Récupération des messages"; // Tchap
 "secrets_recovery_with_key_information_default" = "Accédez à l’historique de vos messages sécurisés et à votre identité de signature croisée pour vérifier d’autres sessions en renseignant votre clé de récupération.";
-"secrets_recovery_with_key_information_verify_device" = "Utilisez votre clé de récupération pour vérifier cet appareil.";
+"secrets_recovery_with_key_information_verify_device" = "Insérez votre Code de Récupération pour vérifier cet appareil."; // Tchap
 "secrets_recovery_with_key_recovery_key_title" = "Saisir";
 "secrets_recovery_with_key_recovery_key_placeholder" = "Saisir la clé de récupération";
-"secrets_recovery_with_key_recover_action" = "Utiliser la clé";
-"secrets_recovery_with_key_invalid_recovery_key_title" = "Impossible d’accéder au coffre secret";
-"secrets_recovery_with_key_invalid_recovery_key_message" = "Vérifiez que vous avez saisi la bonne clé de récupération.";
+"secrets_recovery_with_key_recover_action" = "Utilizer le Code de Récupération"; // Tchap
```
Comment (odelcroi) [inline Riot/Assets/fr.lproj/Vector.strings:1129]: Utiliser

File: Riot/Assets/fr.lproj/Vector.strings [hunk 3]
```diff
@@ -1120,14 +1121,14 @@
 "secrets_recovery_with_passphrase_lost_passphrase_action_part3" = ".";
 "secrets_recovery_with_passphrase_invalid_passphrase_title" = "Impossible d’accéder au coffre secret";
 "secrets_recovery_with_passphrase_invalid_passphrase_message" = "Vérifiez que vous avez saisi la bonne phrase secrète de récupération.";
-"secrets_recovery_with_key_title" = "Clé de récupération";
+"secrets_recovery_with_key_title" = "Récupération des messages"; // Tchap
 "secrets_recovery_with_key_information_default" = "Accédez à l’historique de vos messages sécurisés et à votre identité de signature croisée pour vérifier d’autres sessions en renseignant votre clé de récupération.";
-"secrets_recovery_with_key_information_verify_device" = "Utilisez votre clé de récupération pour vérifier cet appareil.";
+"secrets_recovery_with_key_information_verify_device" = "Insérez votre Code de Récupération pour vérifier cet appareil."; // Tchap
 "secrets_recovery_with_key_recovery_key_title" = "Saisir";
 "secrets_recovery_with_key_recovery_key_placeholder" = "Saisir la clé de récupération";
```
Comment (odelcroi) [inline Riot/Assets/fr.lproj/Vector.strings:1128]: code ?

**Review by odelcroi — CHANGES_REQUESTED** [review]

File: Riot/Assets/en.lproj/Vector.strings (hunk 1 above)
Comment (NicolasBuquet) [reply Riot/Assets/en.lproj/Vector.strings:1582]: Fixed

File: Riot/Assets/fr.lproj/Vector.strings (hunk 2 above)
Comment (NicolasBuquet) [reply Riot/Assets/fr.lproj/Vector.strings:1129]: Fixed

**Pull request merged**